packages/story-editor/src/components/library/panes/shapes/shapePreview.js

```diff
@@ -54,16 +53,14 @@ const Aspect = styled.button`
   border-radius: ${({ theme }) => theme.borders.radius.small};
   background-color: ${({ theme }) => theme.colors.interactiveBg.previewOverlay};
 
-  transition: background-color ${BUTTON_TRANSITION_TIMING};
-
-  &:hover,
-  &:focus,
-  &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR} {
-    background-color: ${({ theme }) =>
-      theme.colors.interactiveBg.secondaryHover};
+  &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR} [role='presentation'],
```

Comment:
Since the Insertion button is not a real button (unless it opens a menu -- in case of media and Saved page templates) then it seemed simpler to just assign a presentational role to it and add the focus outline inside the element that's actually focusable and interactive if the parent is focused.

Alternative could be reworking all the element library to have the `+` icon as a "real" button instead and focus on that only. Then we would need to use the new nested roving tab index hook. Not sure if it's worthwhile but might be more "correct".

Thoughts?

Reply:
Keeping as it is for now for simplicity, we can create a follow-up issue to rework this if necessary. Changing it to a real button will require quite a bit of refactoring.